.github/workflows/ci.yml

```diff
@@ -173,17 +151,26 @@ jobs:
         image: rabbitmq:latest
         ports:
           - 5672:5672
-        options: --health-cmd "rabbitmqctl node_health_check" --health-interval 10s --health-timeout 5s --health-retries 5
+        options: >-
+          --health-cmd "rabbitmqctl node_health_check" 
+          --health-interval 10s 
+          --health-timeout 5s 
+          --health-retries 5
       memcached:
         image: memcached:latest
         ports:
           - 11211:11211
-        options: --health-cmd "timeout 5 bash -c 'cat < /dev/null > /dev/udp/127.0.0.1/11211'" --health-interval 10s --health-timeout 5s --health-retries 5
+        options: >-
```

Comment:
Is this a lot of white space? We should probably remove excess white space here unless it's for a purpose